configs/openimage/openimage_challenge.py

```diff
@@ -0,0 +1,71 @@
+_base_ = [
+    '../_base_/models/faster_rcnn_r50_fpn.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+
+model = dict(
+    roi_head=dict(bbox_head=dict(num_classes=500)),
+    test_cfg=dict(rcnn=dict(score_thr=0.01)))
+
+# dataset settings
+dataset_type = 'OpenImagesChallengeDataset'
+data_root = 'data/OpenImages/'
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+train_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(type='LoadAnnotations', with_bbox=True, normed_bbox=True),
+    dict(type='Resize', img_scale=(1024, 800), keep_ratio=True),
```

Comment:
In which codebase is this setting  `(1024, 800)` used?

Reply:
in [tf](https://github.com/Sense-X/TSD/blob/master/configs/OpenImages_configs/r50-FPN-1x_classsampling/r50-FPN-1x_classsampling.py), setting 1024, 600
in [TSD](https://github.com/Sense-X/TSD/blob/master/configs/OpenImages_configs/r50-FPN-1x_classsampling/r50-FPN-1x_classsampling.py), setting 1024, 800

Parameters are learning from TSD